tests/wot_coap_generation/config/wot_td/instance.json

```diff
@@ -0,0 +1,29 @@
+{
+    "thing": {
```

Comment:
Okay, it gets the title and description from the model, if I get this right. But don't we need some specifier to identify the device itself? The `id` field is probably good for that. Otherwise I am not able to identify the device.

Reply:
Hmm, yeah, you are right. The `id` field, however, needs to be an `anyURI` (see [here](https://www.w3.org/TR/wot-thing-description/#thing)) – maybe we could, alternatively, just insert the `base` URI a second time during the TD creation process at runtime?